src/actions/petAction.js

```diff
@@ -83,3 +83,49 @@ export const getDogBreeds = async (dispatch) => {
     });
   }
 };
+
+// Get Cat Colors
+export const getCatColors = async (dispatch) => {
+  try {
+    dispatch({ type: 'CAT_COLORS_REQUEST' });
+    const response = await axios.get(`/api/v1/cat-colors`, {
+      withCredentials: true,
```

Comment:
You do not have to send `withCredentials: true` for unprotected api endpoints.

Reply:
withCredentials: true was removed for unprotected api endpoints.